### DISCUSSION TITLE
Auto-budgets are not working even after fireing cron jobs

### DISCUSSION THREAD

**Discussion by OvidiuTa** [firefly-iii/firefly-iii#7242] [General]
Hello,

could you please advise what else should I do to have the budgeted amount done automatically for every month?
I set-up a few budgets, enabled crontab jobs (as I am running inside docker but with the docu I was able to do it)..but still the amount is not showed for the budgets in the GUI.

Running the API tells me:

<div class="toolbar"><div class="devtools-separator"></div><div class="devtools-searchbox"><input class="searchBox devtools-filterinput" placeholder="Filter JSON" value=""></div></div><div class="panelContent" id="json-scrolling-panel" tabindex="0">

  |  
-- | --
recurring_transactions |  
job_fired | false
job_succeeded | false
job_errored | false
message | "It has been 3h after since the recurring transactions cron-job has fired. It will not fire now."
auto_budgets |  
job_fired | false
job_succeeded | false
job_errored | false
message | "It has been 3h after since the auto budget cron-job has fired. It will not fire now."

</div>
Thank you!!

**Comment by JC5** [comment]
The cron job only changes things on the first of the [period], so usually the first of the month. Other days, it will fire but change nothing.

**Reply by OvidiuTa** [reply]
Thanks for your reply! I missed this from the documentation _You can't change the trigger dates (like every Tuesday or the 5th of the month). After the amount has been created (on the first of each period or daily) you can freely edit or remove the limit. Firefly III won't stop you._

**Reply by JC5** [reply]
If you want to validate the cron job, run it manually: `--force --date=2023-03-01`.

**Reply by Zyadon7** [reply]
This seems to work for previous months, but not the current month.  Is that correct?

**Reply by JC5** [reply]
What do you mean?
